dnf-behave-tests/features/plugins-core/needs-restarting-conf-files.feature

```diff
@@ -0,0 +1,142 @@
+Feature: Add package to needs-restarting using config files
+
+Background:
+Given I enable plugin "needs_restarting"
+  And I use repository "needs-restarting"
+  And I move the clock backward to "before boot-up"
+  And I execute dnf with args "install wget abcde"
+  And I move the clock forward to "the present"
+  And I use repository "needs-restarting-updates"
+Given I create directory "/etc/dnf/plugins/needs-restarting.d"
+  And I create and substitute file "/etc/dnf/plugins/needs-restarting.d/wget.conf" with
+  """
+  wget
+  """
+
+@bz1810123
+Scenario: restarting needed after updating package in conf files
+Given I execute dnf with args "upgrade wget"
+ When I execute dnf with args "needs-restarting -r"
+ Then the exit code is 1
+  And stdout is
+  """
+  Core libraries or services have been updated since boot-up:
+    * wget
+
+  Reboot is required to fully utilize these updates.
+  More information: https://access.redhat.com/solutions/27943
+  """
+
+@bz1810123
+Scenario: conf files can be more than one
+Given I create and substitute file "/etc/dnf/plugins/needs-restarting.d/abcde.conf" with
+"""
+abcde
+"""
+  And I execute dnf with args "upgrade wget abcde"
+ Then the exit code is 0
```

Comment:
This actually is a setup step to test the `needs-restarting` functionality. You should use this instead of the two above lines:
```
Given I successfully execute dnf with args "upgrade wget abcde"
```

(Or, in this case `Given` is the active step so you should use `And`)

More occurrences of this below.